build.roc

```diff
@@ -136,7 +134,7 @@ copyHostLib! = \osAndArch, rustTargetFolder ->
 
     hostDestPath = "platform/$(prebuiltStaticLibFile osAndArch)"
```

Comment:
Still seeing a lot of camel case. Did you run `roc format --migrate`?

Reply:
Wait, there's an auto formatter for that??! I did it all by hand 😭

Reply:
Thanks for bringing this up @gamebox, could you put a small message in zulip #announcements ? That should increase awareness.